aiohttp_swagger3/swagger.py

```diff
@@ -82,7 +72,7 @@ def __init__(
 
         for ui in uis:
             if ui is not None:
-                self._register_ui(ui)
+                ui._register_ui(self)
```

Comment:
I don't like that change, could you rollback that change?

Reply:
I still think this is a good idea. However, it can be submitted in a separately, which I'll do.